llvm/test/LTO/AArch64/TestInputs/bar.ll

```diff
@@ -0,0 +1,35 @@
+; This file contains the new semantic of the branch-target-enforcement, sign-return-address.
```

Comment:
Nit: in llvm/test/Bitcode/upgrade-branch-protection.ll, you use ';;' for actual comments and ';' for special RUN and CHECK lines. New tests seem to often use ';;' for actual comments, so it's better to use this convention here as well IMHO. Also applies to other newly introduced tests.

Reply:
done.